### PR TITLE
[Merged by Bors] - feat: add lemmas `Disjoint.isCompl_sup_right_of_isCompl_sup_left` (and partner)

### PR DESCRIPTION
**Pull request by ocfnash**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Comment by kbuzzard** [comment]
LGTM

maintainer merge

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Comment by ocfnash** [comment]
The build failure was due to a "No space left on device" error for hoskinson9, so I'll try sending this back to bors in the hopes of getting a different runner.

bors merge

File: Mathlib/Order/ModularLattice.lean [hunk 1]
```diff
@@ -388,23 +388,39 @@ instance (priority := 100) [DistribLattice α] : IsModularLattice α :=
...
-    [IsModularLattice α] {a b c : α} (h : Disjoint b c) (hsup : Disjoint a (b ⊔ c)) :
+theorem disjoint_sup_left_of_disjoint_sup_right [Lattice α] [OrderBot α]
+    [IsModularLattice α] (h : Disjoint b c) (hsup : Disjoint a (b ⊔ c)) :
     Disjoint (a ⊔ b) c := by
   rw [disjoint_comm, sup_comm]
   apply Disjoint.disjoint_sup_right_of_disjoint_sup_left h.symm
   rwa [sup_comm, disjoint_comm] at hsup
 #align disjoint.disjoint_sup_left_of_disjoint_sup_right Disjoint.disjoint_sup_left_of_disjoint_sup_right
 
+theorem isCompl_sup_right_of_isCompl_sup_left [CompleteLattice α] [IsModularLattice α]
```
[the remark this answers is not in the document]
Comment (ocfnash) [reply Mathlib/Order/ModularLattice.lean:412]: Good point, I've implemented this in https://github.com/leanprover-community/mathlib4/pull/7555

**Pull request closed without merging**